src/json-patch-ot-agent.js

```diff
@@ -1,15 +1,21 @@
-if (typeof require !== "undefined" && typeof JSONPatchQueue === 'undefined') {
-	JSONPatchQueue = require('json-patch-queue');
+if(typeof JSONPatchQueue === 'undefined') {
+	if(typeof require !== 'undefined') {
+		var JSONPatchQueue = require('json-patch-queue').JSONPatchQueue;
+	}
+	else {
+		throw new Error('You need to reference JSONPatchQueue before JSONPatchOTAgent');
+	}
 }
+
 /**
  * [JSONPatchOTAgent description]
+ * version: 1.1.0
```

Comment:
Previously it was detecting everything that matches `/version\s*:/`.
In one of our repos we user custom regex to detect that in other files as well.